import-scripts/verify-management-state.sh

```diff
@@ -10,42 +10,79 @@
 # - capture the current database color according to the management database table
 # - exit with error if they differ
 
-unset BLUE_SERVICE_LIST
-unset GREEN_SERVICE_LIST
-declare -a BLUE_SERVICE_LIST
-declare -a GREEN_SERVICE_LIST
-BLUE_SERVICE_LIST+=('cbioportal-backend-public-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-public-beta-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-master-blue')
-BLUE_SERVICE_LIST+=('cbioportal-backend-clickhouse-only-db-blue')
-GREEN_SERVICE_LIST+=('cbioportal-backend-public-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-public-beta-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-master-green')
-GREEN_SERVICE_LIST+=('cbioportal-backend-clickhouse-only-db-green')
-PUBLIC_CLUSTER_INGRESS_NAME='cbioportal-ingress'
-tmp="/data/portal-cron/tmp/import-cron-public"
+declare -a BLUE_SERVICE_LIST=()
+declare -a GREEN_SERVICE_LIST=()
+YQ_BINARY=${YQ_BINARY:-yq}
+
+function read_scalar() {
```

Comment:
this function (and the following function read_array()) is duplicated in the transfer_deployment script. Perhaps a small sourceable module with the YQ functionality could be created separately and sourced into these two scripts (or anywhere else needed, because I think this approach might be very applicable to other scripting)

Reply:
can discuss this more